.github/BOTMETA.yml

```diff
@@ -1038,6 +1038,8 @@ files:
   $modules/proxmox_template.py:
     ignore: skvidal
     maintainers: UnderGreen
+  $modules/proxmox_node_info.py:
```

Comment:
Can you move this up a bit so it fits alphabetically between the other `proxmox_*` modules?